recipes/spyrmsd/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spyrmsd" %}
+{% set version = 0.3.0 %}
```

Comment:
I believe you need quotation marks around the version number.

Reply:
Thanks, I completely missed that!